block/s3/inventory.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3iface"
 	"github.com/treeverse/lakefs/block"
-	inventorys3 "github.com/treeverse/lakefs/inventory/s3"
+	inventorys3 "github.com/treeverse/lakefs/cloud/aws/s3inventory"
```

Comment:
why is the import named in reverse? what's wrong with `s3inventory`?

Reply:
it was like this before and I didn't want to bloat the diff. But now that you've noticed it I must